fix: mention of jenkins-infra/helpdesk for Infrastructure project issues (#4816)
jenkins-infra/jenkins.io: content/participate/report-issue.adoc

* For *Project* select:
** _Jenkins_ for general issues with Jenkins,
** _Security Issues_ if you want to report a security issue privately

> * For _Infrastructure_, if you're reporting a bug with a Jenkins service run by the Jenkins project, link: open an issue in the https://github.com/jenkins-infra/helpdesk/issues/new/choose[dedicated helpdesk on Github].

Author: ```suggestion
* For _Infrastructure_, if you're reporting a bug with a Jenkins service https://www.jenkins.io/projects/infrastructure/[run by the Jenkins project], link: open an issue in the https://github.com/jenkins-infra/helpdesk/issues/new/choose[dedicated helpdesk on Github].
```

> * For _Infrastructure_, if you're reporting a bug with the Jenkins web site or any other services run by the Jenkins project, link: open an issue in the https://github.com/jenkins-infra/helpdesk/issues/new/choose[dedicated helpdesk on Github].

Reviewer: I wouldn't mention the website as 99% of issues belong to the website and not infra.

```suggestion
* For _Infrastructure_, if you're reporting a bug with a Jenkins service run by the Jenkins project, link: open an issue in the https://github.com/jenkins-infra/helpdesk/issues/new/choose[dedicated helpdesk on Github].
```

> * For _Infrastructure_, if you're reporting a bug with https://www.jenkins.io/projects/infrastructure/[a Jenkins service run by the Jenkins project], link: open an issue in the https://github.com/jenkins-infra/helpdesk/issues/new/choose[dedicated helpdesk on Github].

Author: ```suggestion
* For _Infrastructure_, if you're reporting a bug with https://www.jenkins.io/projects/infrastructure/[a Jenkins service run by the Jenkins project], open an issue in the https://github.com/jenkins-infra/helpdesk/issues/new/choose[dedicated helpdesk on Github].
```
